src/login/LoginMain/LoginMain.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Router, Route } from "react-router-dom";
+import { createBrowserHistory } from "history";
+
+import FetchingContext from "components/FetchingContext";
+import SplashContainer from "containers/Splash";
+import NotificationsContainer from "containers/Notifications";
+import FooterContainer from "containers/Footer";
+
+import "style/base.scss";
+import "./LoginMain.scss";
+
+export const history = createBrowserHistory();
+
+class Main extends Component {
```

Comment:
I know I have already merged in LoginMain, but it just struck me now that this component loads the app so what if we called it something like LoginAppLoader or AppLoader...

In fact, would we be able to re-use the `AppLoader`, `Header` and `Footer` for all the apps and get all the app specific stuff later? 

So we load:
```
<AppLoader>
   <Header />
      [App specific config, components and actions]
   <Footer />
</AppLoader> 
```